store/src/commonMain/kotlin/org/mobilenativefoundation/store/store5/impl/RealStoreBuilder.kt

```diff
@@ -33,19 +33,20 @@ fun <Key : Any, Network : Any, Output : Any> storeBuilderFromFetcherSourceOfTrut
     sourceOfTruth: SourceOfTruth<Key, Network, Output>,
     memoryCache: Cache<Key, Output>,
 ): StoreBuilder<Key, Output> =
-    RealStoreBuilder(fetcher, sourceOfTruth, memoryCache)
+    RealStoreBuilder<Key, Network, Output, Network>(fetcher, sourceOfTruth, memoryCache)
+
+fun <Key : Any, Network : Any, Output : Any, Local: Any> storeBuilderFromFetcherSourceOfTruthMemoryCacheAndConverter(
```

Comment:
I followed the existing conventions for naming this function, but just out of curiosity: why not use a single function with default arguments?